tests/test_validators.py

```diff
@@ -48,3 +52,257 @@ def test_value_valid(self):
+            self.validate(self.empty_list)
+
+
+class ValidateListOfIntegersTestCase(unittest.TestCase):
+    def setUp(self):
+        self.field_name = fake.word()
+        self.correct_value = [1, 2, 3]
+        self.incorrect_value = [1.2, 0., 1.0]
+        self.min_length = 1
+        self.max_length = 3
+
+    def validate(self, value):
+        return validators.validate_list_of_integers(
+            field_name=self.field_name,
+            value=value,
+            min_length=self.min_length,
+            max_length=self.max_length,
+        )
+
+    def test_that_correct_value_will_not_raise_exception(self):
+        self.validate(self.correct_value)
+
+    def test_that_incorrect_value_will_raise_exception(self):
+        with self.assertRaises(exceptions.FieldError):
+            self.validate(self.incorrect_value)
+
+
+class ValidateListOfFloatsTestCase(unittest.TestCase):
+    def setUp(self):
+        self.field_name = fake.word()
+        self.correct_value = [1.2, 3.4, 5.6]
+        self.incorrect_value = [1, 2, 3]
+        self.min_length = 1
+        self.max_length = 3
+
+    def validate(self, value):
+        return validators.validate_list_of_floats(
+            field_name=self.field_name,
+            value=value,
+            min_length=self.min_length,
+            max_length=self.max_length,
+        )
+
+    def test_that_correct_value_will_not_raise_exception(self):
+        self.validate(self.correct_value)
+
+    def test_that_incorrect_value_will_raise_exception(self):
+        with self.assertRaises(exceptions.FieldError):
+            self.validate(self.incorrect_value)
+
+
+class ValidateEnumTestCase(unittest.TestCase):
+    def setUp(self):
+        self.correct_value = 'Test'
+        self.incorrect_value = 'Wrong_test'
+        self.field_name = fake.word()
+
+    class EnumTest(StringEnum):
+        Test = enum.auto()
+
+    def validate(self, value, enum_class):
+        return validators.validate_enum(
+            field_name=self.field_name,
+            value=value,
+            enum_class=enum_class,
+        )
+
+    def test_that_correct_enum_value_will_not_raise_exception(self):
+        self.validate(
+            value=self.correct_value,
+            enum_class=self.EnumTest
+        )
+
+    def test_that_incorrect_enum_value_will_raise_exception(self):
+        with self.assertRaises(exceptions.FieldError):
+            self.validate(
+                value=self.incorrect_value,
+                enum_class=self.EnumTest,
+            )
+
+
+class ValidateTaskTypeWithMetaParametersTestCase(unittest.TestCase):
+    def setUp(self):
+        self.field_name = fake.word()
+        self.valid_pairs = [
+            (tasks.TaskType.Blender, tasks.BlenderScriptPackage),
+            (tasks.TaskType.LuxRender, tasks.LuxRenderScriptPackage),
+        ]
+        self.blender_script_package = BlenderScriptPackageFactory()
+        self.correct_value = {
```

Comment:
What about `LuxRender`? It might be worth checking as well ;)

Reply:
Added